### PR TITLE
fix packit leftovers

### PR DESCRIPTION
**Pull request by marusinm**
Resolves Issues #2 #3 #4 #5.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 CONT_HOME := /opt/app-root/src
 
-PACKIT_SERVICE_IMAGE := docker.io/marusinm/release-bot:latest
+RELEASE_BOT_IMAGE := docker.io/marusinm/release-bot:latest
```
Comment (rpitonak) [inline Makefile]: shouldn't this be `docker.io/usercont/release-bot:latest`?

File: secrets/README.md [hunk 2]
```diff
@@ -4,32 +4,13 @@
...
-    │   ├── fullchain.pem
-    │   ├── httpd-packit.conf
-    │   ├── id_rsa
-    │   ├── id_rsa.pub
-    │   ├── packit-service.yaml
-    │   ├── private-key.pem
-    │   ├── privkey.pem
-    │   └── ssh_config
+    │   ├── conf.yaml
+    │   ├── private-key.pema
```
Comment (rpitonak) [inline secrets/README.md]: typo: `.pema` -> `.pem`

File: secrets/template/conf.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+repository_name: name
```
Comment (rpitonak) [inline secrets/template/conf.yaml:1]: This is not a correct template, we will need here fields for GitHub app authentication. But feel free to change it later, when the functionality is actually implemented in `release-bot`.

**Review by rpitonak — CHANGES_REQUESTED** [review]
I left a few notes.

**Comment by marusinm** [comment]
I pushed changes but I haven't changed the `conf.yaml` yet. As you suggest I leave it for now and edit when I will need Github app flags there.

**Review by rpitonak — APPROVED** [review]
LGTM

**Pull request merged**